modern/src/reports/RouteReportPage.js

```diff
@@ -26,35 +27,64 @@ const Filter = ({ setItems }) => {
 };
 
 const RouteReportPage = () => {
+  const distanceUnit = useAttributePreference('distanceUnit');
+
+  const columns = [{
+    headerName: t('positionFixTime'),
+    field: 'fixTime',
+    type: 'dateTime',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'fixTime'),
+  }, {
+    headerName: t('positionLatitude'),
+    field: 'latitude',
+    type: 'number',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'latitude'),
+  }, {
+    headerName: t('positionLongitude'),
+    field: 'longitude',
+    type: 'number',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'longitude'),
+  }, {
+    headerName: t('positionSpeed'),
+    field: 'speed',
+    type: 'number',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'speed'),
+  }, {
+    headerName: t('positionAddress'),
+    field: 'address',
+    type: 'string',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'address'),
+  }, {
+    headerName: t('positionIgnition'),
+    field: 'ignition',
+    type: 'boolean',
+    flex: 1,
+    valueGetter: ({ row }) => row.attributes.ignition,
+    valueFormatter: ({ value }) => value ? 'Yes' : 'No',
```

Comment:
OK, I think we are almost there, but we shouldn't have hardcoded values like this. By the way, instead of `formatPosition` can we use specific formatters directly?